notebooks/centrality/Centrality.ipynb

```diff
@@ -0,0 +1,771 @@
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_8c7df_row2_col0\" class=\"data row2 col0\" >0.148142</td>\n",
+       "                        <td id=\"T_8c7df_row2_col1\" class=\"data row2 col1\" >1430</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_8c7df_row3_col0\" class=\"data row3 col0\" >0.148142</td>\n",
+       "                        <td id=\"T_8c7df_row3_col1\" class=\"data row3 col1\" >1431</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_8c7df_row4_col0\" class=\"data row4 col0\" >0.144147</td>\n",
+       "                        <td id=\"T_8c7df_row4_col1\" class=\"data row4 col1\" >645</td>\n",
+       "            </tr>\n",
+       "    </tbody></table><style  type=\"text/css\" >\n",
+       "</style><table id=\"T_1587f_\" style='display:inline'><caption>PageRank</caption><thead>    <tr>        <th class=\"col_heading level0 col0\" >pagerank</th>        <th class=\"col_heading level0 col1\" >vertex</th>    </tr></thead><tbody>\n",
+       "                <tr>\n",
+       "                                <td id=\"T_1587f_row0_col0\" class=\"data row0 col0\" >0.004183</td>\n",
+       "                        <td id=\"T_1587f_row0_col1\" class=\"data row0 col1\" >78</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_1587f_row1_col0\" class=\"data row1 col0\" >0.003771</td>\n",
+       "                        <td id=\"T_1587f_row1_col1\" class=\"data row1 col1\" >33</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_1587f_row2_col0\" class=\"data row2 col0\" >0.002800</td>\n",
+       "                        <td id=\"T_1587f_row2_col1\" class=\"data row2 col1\" >34</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_1587f_row3_col0\" class=\"data row3 col0\" >0.002387</td>\n",
+       "                        <td id=\"T_1587f_row3_col1\" class=\"data row3 col1\" >281</td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                                <td id=\"T_1587f_row4_col0\" class=\"data row4 col0\" >0.002373</td>\n",
+       "                        <td id=\"T_1587f_row4_col1\" class=\"data row4 col1\" >294</td>\n",
+       "            </tr>\n",
+       "    </tbody></table>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "print_centrality(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now see a large discrepancy in which each algorithm thinks is the most important vertex.\n",
```

Comment:
Seems like this sentence could be reworded, but at a minimum: "thinks is the" -> "thinks it is the"

Reply:
reworded